hippo4j-common/src/test/java/cn/hippo4j/common/MockitoTests.java

```diff
@@ -0,0 +1,33 @@
+package cn.hippo4j.common;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.anyInt;
+
+/**
+ * Mockito 集成测试
```

Comment:
Please modify according to the historical code comment specification.

Reply:
> Please modify according to the historical code comment specification.

done.